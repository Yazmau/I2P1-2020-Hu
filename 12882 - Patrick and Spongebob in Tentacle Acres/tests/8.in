63 22
55 27 94 82 20 68 84 3 26 76 15 33 14 84 36 7 94 45 62 64 100 63
63 43 40 93 61 87 9 12 14 20 92 84 37 72 65 70 79 23 84 60 99 43
17 75 66 25 73 44 92 86 10 19 9 71 95 79 9 69 22 85 48 30 8 93
50 70 24 48 61 23 92 92 50 70 63 18 59 91 53 23 80 93 39 11 74 39
100 46 90 60 33 54 16 20 37 10 90 53 2 46 64 99 50 8 51 72 79 21
75 85 4 54 71 47 65 68 92 83 75 84 66 31 34 7 27 34 13 27 15 44
45 37 10 51 35 21 86 87 85 17 12 55 73 2 23 26 10 53 66 3 65 35
77 66 13 45 51 53 39 25 39 43 83 31 78 100 5 6 67 70 54 81 80 66
84 69 58 86 73 57 25 88 53 62 89 97 47 15 42 75 28 71 37 14 7 18
9 45 58 27 63 50 28 41 30 21 82 2 84 93 22 11 70 17 63 57 4 13
3 9 79 43 2 77 86 78 50 2 25 50 13 24 21 22 57 45 30 4 1 34
75 21 50 8 78 27 75 88 57 77 49 68 79 70 83 4 76 6 42 17 28 15
87 29 85 42 1 97 77 12 43 34 37 37 84 42 92 28 65 64 49 21 65 70
72 77 22 70 69 98 61 65 42 13 59 23 100 2 74 94 13 67 48 51 77 53
96 63 89 74 38 32 2 41 40 57 76 89 44 37 4 22 79 33 41 43 53 97
92 34 26 15 12 23 58 96 47 58 58 62 95 16 69 64 54 77 88 17 90 77
59 37 10 15 10 57 71 97 65 92 99 55 40 93 62 64 39 86 48 34 23 64
44 66 39 8 80 54 67 60 23 50 33 80 85 100 76 82 43 47 63 90 12 40
20 74 97 96 50 15 59 7 73 87 99 5 38 73 68 31 41 32 93 26 41 20
59 49 16 98 49 90 76 45 28 73 82 8 47 53 83 81 8 40 18 93 84 19
84 74 63 85 76 44 54 67 96 68 99 10 34 99 37 93 55 44 90 6 99 14
71 26 79 55 79 38 2 25 68 14 48 75 92 42 30 12 37 44 68 32 34 44
6 57 32 24 68 95 74 52 86 72 85 75 100 78 96 64 39 20 18 62 98 36
75 20 98 4 38 46 43 24 22 6 13 66 21 35 47 51 52 90 47 7 74 64
40 60 96 2 3 4 30 29 23 34 100 74 37 73 54 71 92 49 97 12 62 56
48 99 97 87 44 88 99 30 28 14 19 63 70 44 57 75 76 41 6 12 40 32
17 50 76 25 88 30 22 94 13 26 89 99 45 35 39 99 60 25 9 52 23 38
70 94 45 16 56 53 91 26 50 39 44 71 52 11 33 15 50 17 30 4 73 99
91 31 8 25 16 64 60 22 18 34 13 77 72 24 79 73 22 33 94 50 81 85
6 61 9 89 51 52 54 7 35 52 56 87 59 12 87 68 10 65 67 25 34 24
1 97 72 22 40 35 60 5 29 50 67 52 55 23 35 47 16 78 31 80 38 16
61 31 11 24 18 59 26 21 17 47 8 30 52 15 84 58 17 88 54 68 98 23
88 9 82 14 28 69 42 81 43 5 44 3 74 36 28 82 8 77 62 31 45 7
25 36 46 89 11 28 74 50 63 94 2 79 30 74 94 32 54 70 70 27 82 86
75 80 20 10 19 62 97 67 24 44 61 33 98 1 69 11 100 1 58 17 51 53
33 22 19 87 85 52 100 76 58 22 46 81 54 81 5 98 15 19 30 90 19 78
21 4 40 63 16 16 4 53 22 68 65 38 78 11 58 5 100 86 97 43 87 81
59 81 94 95 63 75 66 76 39 16 81 68 37 88 3 50 100 13 36 63 25 70
79 83 38 99 42 40 34 27 83 87 90 73 16 100 97 99 16 50 82 10 88 73
45 92 100 26 14 5 63 38 94 23 54 51 55 21 78 46 57 96 45 8 19 85
97 9 34 36 33 16 31 73 30 41 7 93 43 1 58 77 5 78 82 64 32 84
27 60 16 77 74 49 45 41 63 17 20 39 69 57 40 23 82 55 97 6 80 66
60 53 9 62 77 29 67 5 44 73 22 98 5 24 61 54 92 59 13 74 89 45
46 42 93 2 3 78 39 59 66 43 12 41 2 39 48 9 60 6 98 94 86 13
92 59 59 55 77 21 25 47 85 19 41 8 32 37 3 41 71 36 43 73 94 97
41 68 68 38 14 44 23 15 25 96 17 46 7 25 32 28 75 56 62 25 7 26
62 88 80 4 73 73 14 87 26 29 2 17 51 46 15 62 53 36 22 76 56 53
77 83 37 26 19 86 6 100 82 64 81 40 72 80 90 51 88 29 60 6 50 39
84 75 46 72 2 22 93 51 18 25 50 53 92 49 34 19 37 51 64 86 31 100
83 87 85 17 23 7 41 55 48 24 20 8 79 50 29 92 90 35 22 45 20 3
10 92 35 85 21 78 93 56 100 60 39 79 98 39 13 26 20 40 16 74 1 37
10 17 43 30 50 84 28 26 17 46 18 15 34 48 72 89 99 43 14 56 66 24
85 13 73 21 81 18 34 77 57 15 36 72 34 93 52 72 89 74 24 25 72 59
36 18 37 29 16 22 35 9 51 32 55 60 39 44 88 3 7 83 23 12 80 7
24 96 64 77 19 67 94 100 5 99 55 81 84 1 47 89 90 62 2 37 99 83
44 7 2 17 54 58 88 70 38 30 36 59 18 81 4 20 18 52 95 3 9 56
86 53 44 45 7 43 88 94 9 10 63 50 50 68 95 89 2 80 10 66 99 42
91 27 100 25 51 91 64 55 30 63 81 69 84 73 91 60 80 35 73 97 84 78
88 82 10 93 27 15 9 74 76 29 54 96 97 87 56 99 96 56 36 94 27 78
9 1 41 14 58 57 20 25 79 70 36 93 60 92 70 66 90 26 2 29 27 50
97 89 94 35 61 81 29 46 36 7 81 97 89 44 94 79 64 69 29 19 79 100
4 28 92 1 21 94 4 49 11 4 11 63 91 84 29 42 100 70 86 64 40 57
91 82 53 93 15 9 33 79 59 75 86 20 31 52 10 31 35 54 44 6 59 65
94
51 52 1 11
54 56 3 10
43 62 10 12
21 50 22 22
6 42 12 18
5 29 5 12
11 36 18 20
10 16 9 15
18 36 15 19
27 55 5 13
20 61 17 22
20 27 4 9
56 59 14 22
19 20 19 22
59 61 11 20
42 57 18 22
62 62 20 22
11 60 17 18
39 53 12 19
2 43 17 18
11 19 12 19
12 19 21 22
55 63 8 13
56 60 17 20
45 63 6 17
46 56 21 21
57 57 15 19
29 47 12 15
17 54 18 19
37 60 7 12
10 14 19 19
53 60 14 19
41 51 9 13
45 61 19 19
44 51 12 18
57 61 4 5
51 63 11 20
30 54 20 21
15 44 21 22
34 34 20 20
20 41 13 13
28 31 6 14
31 61 6 20
23 42 9 17
45 62 3 4
18 57 8 14
12 50 16 21
26 56 10 19
21 52 12 17
53 57 6 12
21 59 6 8
22 57 17 22
1 1 2 12
23 24 19 21
15 45 20 20
21 59 2 6
21 47 7 19
47 50 16 21
24 37 18 20
40 40 7 22
26 36 11 14
34 48 9 18
63 63 15 21
41 56 8 21
13 46 7 14
9 56 1 9
35 42 14 16
25 53 22 22
12 31 11 12
10 30 8 14
9 14 3 3
25 44 17 22
10 11 20 22
18 34 22 22
32 45 17 22
26 41 21 21
46 61 7 17
33 53 12 14
39 41 16 16
21 39 1 3
12 25 10 15
57 60 22 22
22 50 6 19
22 57 3 21
3 30 8 17
47 52 8 20
16 36 15 17
61 63 15 21
38 52 16 19
29 48 2 16
63 63 15 17
26 27 13 20
30 48 3 21
3 7 6 11
