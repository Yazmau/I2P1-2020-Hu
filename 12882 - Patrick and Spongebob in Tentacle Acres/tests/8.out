100
100
99
100
100
100
97
100
100
100
100
99
99
93
99
100
64
100
100
100
100
97
100
97
100
99
95
100
98
100
63
99
100
95
98
93
100
98
99
27
100
91
100
100
100
100
100
100
100
100
100
100
94
98
94
100
100
92
97
96
99
100
59
100
100
100
100
100
100
100
85
100
57
99
100
98
100
100
99
99
100
78
100
100
100
100
100
100
100
100
35
99
100
92
